app/models/ui/Sequence.scala

```diff
@@ -16,4 +16,5 @@
 
 package models.ui
 
-case class Sequence(text: Text, hint: Option[Text], options: Seq[Text], body: Seq[UIComponent], errorMsgs: Seq[ErrorMsg]) extends SequenceFormComponent
+case class Sequence(text: Text, hint: Option[Text], options: Seq[Text], exclusive: Boolean, body: Seq[UIComponent], errorMsgs: Seq[ErrorMsg])
```

Comment:
can see your storing the exclusive as head of options, addition of SequenceExclusive would separate out the exclusive option and simplify the view code. comment only though

Reply:
I've implemented ExclusiveSequence; not a minor update though.